### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kumilange**
The draft for `README.md`. 
I wasn't sure what to write these 3 items below

> ## Issues
> ## Backlog
> ## Contributions

Feel free to add more!

### CODE REVIEW

**Review by tomosm — COMMENTED** [review]
@kumiko-haraguchi Please add the way to run the server and client

**Comment by kumilange** [comment]
@tomosm Thank you for your comment! I fixed it! :)

File: README.md [hunk 1]
```diff
@@ -1,63 +1,74 @@
...
-- Station data is collected by scraping [Wikipedia](https://en.wikipedia.org/wiki/List_of_railway_stations_in_Japan:_A) for a list of station names in Japan and then getting their latitude and longitude from the Google geocoding API.
-- You can scrape the data again yourself by inserting a Google API key in `scrape-station-names.js` on line 8 and running `node scrape-station-names.js`.
+- [AirMap API](https://developers.airmap.com/)
+- Postgres, knex
+- Node/Express
+- Chai
+- React.js([create-react-app](https://github.com/facebookincubator/create-react-app), [react-google-maps](https://github.com/tomchentw/react-google-maps)), Redux
+- google-material-ui
+- scraping
+  - Station data is collected by scraping [Here](https://en.wikipedia.org/wiki/List_of_railway_stations_in_Japan:_A) for a list of station names in Japan and then getting their latitude and longitude from the Google geocoding API.
```
Comment (billfeng) [inline README.md]: Why did you change `Wikipedia` to `Here` ?

**Review by billfeng — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kumilange) [reply README.md]: @bill-feng Oops! Sorry, didn't realize it was an actually wikipedia page! I changed here back to wikipedia! 👍

Comment (billfeng) [reply README.md]: Cool, thanks! :)

**Review by billfeng — APPROVED** [review]
